__tests__/Map.ts

```diff
@@ -447,6 +447,26 @@ describe('Map', () => {
     expect(m.get(c)).toBe('c');
   });
 
+  it('supports Symbols as object constructor keys', () => {
+    const a = Symbol.for('a');
+    const b = Symbol('b');
+    const c = Symbol('c');
+    const m = Map<symbol, string>({
+      // @ts-expect-error -- typescript definition does not handle symbol for now
```

Comment:
This should work - ideally you can add something to the ts-tests as well to assert this

https://www.typescriptlang.org/play?#code/CYUwxgNghgTiAEAzArgOzAFwJYHtXwxAGcMAeAaXhAA9DVgj4SYtUBzeAHyYE8BbAEY4IAGngA1AHwAKHAIBWALngBveAG0ACvFbxyAXQD8y8fAC+ASmUBZKAAcKYqQG4AsACgPYPCSbwAvPAAyvxCENIA5DgA1hEWXj4YBAHBocKRqDhJMXEeHgD0+cEAFjjIEMA6qIggMDb2pBFQEVwEPHYgOIhMYqjIgrWSCai+cETlSYGEJNJqUMoATGLqRPrKAMzm8Z7uYxMAdGwgGJHN23sQGIfH0kTbhfAAAhhEALQ0HZjvMDA4MB4XK5HE4RAS5dwPZ5vD7gDDfX7-XbEA7A6QYbZAA